mdx-components.tsx

```diff
@@ -0,0 +1,29 @@
+import type { MDXComponents } from 'mdx/types'
```

Comment:
These are `MDX` components. [MDX](https://mdxjs.com) is essentially markdown supercharged with JSX, allowing us to use React components in our docs.